docs/example/tipBoxDocs.vue

```diff
@@ -51,7 +64,57 @@
             <tip-box type="definition">
                 definition
             </tip-box>
+            <tip-box type="info" icon=":rocket:">
+                info but with rocket
+            </tip-box>
+            <tip-box type="info" color="white" background-color="red">
+                info but with different colors
+            </tip-box>
+            <tip-box color="white" background-color="blue" border-color="darkblue">
+                default with custom colors
+            </tip-box>                       
+            <box background-color="white" border-color="grey" border-left-color="blue">
+                empty box with left border
+            </box>
         </doc-code>
+        <doc-table name="TipBox Options">
+          <div>
+            <p>backgroundColor</p>
+            <p><code>String</code></p>
+            <p><code>null</code></p>
+            <p>Color of the background.</p>
+          </div>
+          <div>
+            <p>borderColor</p>
+            <p><code>String</code></p>
+            <p><code>null</code></p>
+            <p>Color of the entire border.</p>
+          </div>
+          <div>
+            <p>borderLeftColor</p>
+            <p><code>String</code></p>
+            <p><code>null</code></p>
+            <p>Color of the left border.</p>
```

Comment:
Leave description blank unless it adds value.
- Change this to "Overrides \`border-color\` for the left border."
- Also keep for `color`.